frontend/src/components/ClickableStarRating.tsx

```diff
@@ -0,0 +1,30 @@
+// /src/components/ClickableStarRating.tsx
+import React, { useState } from 'react';
+import Star from './Stars';
+
+interface ClickableStarRatingProps {
+  onRatingChange: (newRating: number) => void;
+}
+
+const ClickableStarRating: React.FC<ClickableStarRatingProps> = ({ onRatingChange }) => {
+  const [rating, setRating] = useState<number | null>(null);
+
```

Comment:
here again, just change it to 'function ClickableStarRatingProps()' to keep it consistent with our code